assets/app/view/game/corporation.rb

```diff
@@ -82,6 +82,7 @@ def render
         extras = []
         if @game.corporation_show_loans?(@corporation)
           extras.concat(render_loans) if @game.total_loans&.nonzero?
+          extras.concat(render_interest) if @game.class::SHOW_INTEREST && @game.total_loans&.nonzero?
```

Comment:
Why did you go with a constant rather than a method?

Reply:
Also, don't repeat if @game.total_loans&.nonzero?. Put both line 84 and 85 within that if statement.

Reply:
1- based on how coupled the previous logic was, i assumed it will be easer to follow if render_loans and render_interest were separated, rather then introduce the logic inside render_loans. LMK if you think i should change it
2- I thought it would be cleaner and more consistent to have a visual toggle as a constant

Reply:
1 - Okay
2 - Since corporation_show_loans? is already a method, I think we should make SHOW_INTEREST constant into a method too. I've found the constants often need to get refactored into a method anyhow as there is nuance in many titles.